map/src/map/util/TrackLayerProvider.js

```diff
@@ -183,6 +183,12 @@ function getPointGeoProfile(point, points) {
     }
 }
 
+export function getPointLatLon(point) {
+    let lat = point.lat ? point.lat : point.ext.lat;
+    let lon = point.lon ? point.lon : point.ext.lon;
+    return lat && lon ? { lat: lat, lon: lon } : null;
```

Comment:
It could be simplified.

const lat = point.lat ?? point.ext.lat;
const lon = point.lon ?? point.ext.lon;
return lat && lon ? { lat, lon } : null;

In any case, lat/lon could be 0,0 and it is still correct lat/lon...